packages/rollup-full-node/test/app/test-web-rpc-handler.spec.ts

```diff
@@ -208,4 +208,56 @@ describe('TestHandler', () => {
       )
     })
   })
+
+  describe('the sendTransaction endpoint', () => {
+    let testRpcServer
+    let httpProvider
+    let wallet
+
+    beforeEach(async () => {
+      testRpcServer = new FullnodeRpcServer(testHandler, host, port)
+      testRpcServer.listen()
+      httpProvider = new ethers.providers.JsonRpcProvider(baseUrl)
+      wallet = getWallets(httpProvider)[0]
+    })
+
+    afterEach(async () => {
+      await testRpcServer.close()
+    })
+
+    it('should run the transaction', async () => {
+      const storageKey = add0x('01'.repeat(32))
+      const storageValue = add0x('02'.repeat(32))
+      const executionManagerAddress = await httpProvider.send(
+        'ovm_getExecutionManagerAddress',
+        []
+      )
+      const factory = new ContractFactory(
+        SimpleStorage.abi,
+        SimpleStorage.bytecode,
+        wallet
+      )
+      const simpleStorage = await factory.deploy()
+      const transactionData = await simpleStorage.interface.functions[
+        'setStorage'
+      ].encode([executionManagerAddress, storageKey, storageValue])
+      const transaction = {
+        nonce: 0,
+        to: simpleStorage.address,
+        gasPrice: 0,
+        gasLimit: 0,
+        value: 0,
+        data: transactionData,
+      }
```

Comment:
Again, it seems like we shouldn't be allowing transactions which are missing a `from` field.  I could be wrong, but for example [here](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_sendtransaction) does not list `from` as optional.

Reply:
Ah, good catch! Updated: -> [Add from address to the `sendTransactionEndpoint` payload](https://github.com/ethereum-optimism/optimism-monorepo/pull/87/commits/5d2c35f5828f93bde9cf194ef50f8c366563e8ad)